ceph.spec.in

```diff
@@ -466,7 +488,17 @@ Requires:	librbd1 = %{epoch}:%{version}-%{release}
 Requires:	python-rados = %{epoch}:%{version}-%{release}
 Obsoletes:	python-ceph < %{epoch}:%{version}-%{release}
 %description -n python-rbd
-This package contains Python libraries for interacting with Cephs RADOS
+This package contains Python 2 libraries for interacting with Cephs RADOS
```

Comment:
I specifically made these changes only in the description and not the summary. So it looks like you think the summaries also all should be changed?